Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: concourse
-version: 8.5.1
-appVersion: 5.7.1
+version: 8.6.0
+appVersion: 5.8.0
```

Comment:
No need to bump these anymore. It's done in CI when we ship the chart. `master` on this repo behaves the same as `concourse/concourse`.